sdk/keyvault/Azure.Security.KeyVault.Keys/tests/JsonWebKeyTests.cs

```diff
@@ -340,7 +340,7 @@ private static bool HasPrivateKey(JsonWebKey jwk)
 
         private class JsonWebKeyComparer : IEqualityComparer<JsonWebKey>
         {
-            internal static readonly IEqualityComparer<JsonWebKey> Instance = new JsonWebKeyComparer();
+            internal static readonly IEqualityComparer<JsonWebKey> s_instance = new JsonWebKeyComparer();
```

Comment:
Non-private statics should not start with "s_" either. They are exposed to code outside the class.

Reply:
Talked offline and while it doesn't look great in code, it's good to be consistent. I can always have a get-accessor for the field if I feel strongly about it. It will likely be inlined.